catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -562,6 +557,18 @@ private SourceResponse cloneResponse(SourceResponse sourceResponse) {
           sourceResponse.getHits(),
           sourceResponse.getProperties());
     }
+
+    private Function<Result, Result> cloneResult =
+        result -> {
+          Metacard metacard = result.getMetacard();
+          Metacard clonedMetacard = new MetacardImpl(metacard, metacard.getMetacardType());
+          Result clonedResult = new ResultImpl(clonedMetacard);
+
+          ((ResultImpl) clonedResult).setDistanceInMeters(result.getDistanceInMeters());
```

Comment:
❓ This seems like a related, but different fix. Was the `distanceInMeters` just being dropped?

Reply:
Correct